src/routes.js

```diff
@@ -76,7 +42,15 @@ export default function getRoutes(
 			const queryResponses$ = proxyApiRequest$(request);
 			queryResponses$.subscribe(
 				queryResponses => {
-					reply(JSON.stringify(queryResponses)).type('application/json');
+					const response = reply(JSON.stringify(queryResponses))
+						.type('application/json');
+
+					// special case - login requests need to be tracked
+					const loginResponse = queryResponses.find(r => r.login);
```

Comment:
Yeah I don't really like having the logic to parse this special case in here, so I'll move it.

I think the 'cleanest' way that allows for future extensibility is to change `reply.track(response, 'login', ...)` to `reply.track(response, 'api', queryResponses)`, which would be a general API response tracking handler that would parse the `queryResponses` for login-specific tracking, and in the future it could be extended to track other kinds of API responses. Once I set it up, I'll see whether it's sufficient to just provide the `queryResponses` or if more info is needed from the route handler.